blocks/commerce-b2b-negotiable-quote-template/commerce-b2b-negotiable-quote-template.js

```diff
@@ -194,21 +196,43 @@ export default async function decorate(block) {
                     vat_id: formValues.vatId,
                   };
 
-                  progressSpinner.removeAttribute('hidden');
-                  shippingInformation.setAttribute('hidden', true);
-                  addQuoteTemplateShippingAddress({
-                    templateId: quoteTemplateId,
-                    shippingAddress: {
-                      address: {
-                        ...addressInput,
-                        additionalInput: additionalAddressInput,
+                  const createCustomerAddressInput = {
+                    city: formValues.city,
+                    company: formValues.company,
+                    countryCode: formValues.countryCode,
+                    defaultBilling: !!formValues.defaultBilling || false,
+                    defaultShipping: !!formValues.defaultShipping || false,
+                    fax: formValues.fax,
+                    firstname: formValues.firstName,
+                    lastname: formValues.lastName,
+                    middlename: formValues.middlename,
+                    postcode: formValues.postcode,
+                    prefix: formValues.prefix,
+                    region: regionCode ? {
+                      regionCode,
+                      regionId: regionIdNumber,
+                    } : undefined,
+                    street: streetInputValues,
+                    suffix: formValues.suffix,
+                    telephone: formValues.telephone,
+                    vatId: formValues.vatId,
+                  };
+
```

Comment:
Do we want to add progress spinner before API call to consistent with negotiable quote?